.gitignore

```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+/public/projects.json
```

Comment:
I don't think this is needed as `projects.json` will live inside the /dist/ folder now?

Reply:
I think that would be abusing vite too much. I'd still continue to generate it into ./public, gitignored, and rely on vite to create the whole ./dist folder

Reply:
Run build, same as before
https://github.com/oasisprotocol/playground/blob/main/package.json#L8

Reply:
Makes sense :+1: